Rakefile

```diff
@@ -111,7 +111,7 @@ namespace :build do
       env_vars = ''
       env_vars += 'MOOC_LOCALIZE=1 ' if CDO.localize_apps
       env_vars += 'MOOC_DIGEST=1 ' unless rack_env?(:development)
-      RakeUtils.system "#{env_vars} grunt"
+      RakeUtils.system "#{env_vars} grunt build"
```

Comment:
Maybe we can keep this a grunt build if we also add an explicit clean to our CI process.